.github/workflows/container.yml

```diff
@@ -11,6 +11,7 @@ jobs:
   build-and-push:
     name: Build and Push to DockerHub
     runs-on: ubuntu-latest
+    if: github.repository == 'algorand/go-algorand'
```

Comment:
This breaks custom images.

(by providing credentials, you can publish custom/fork images to docker)